### ISSUE TITLE
Using ExecTask with environment variables

### ISSUE THREAD

**Issue by jawira** [phingofficial/phing#1253]
Hi, I'm trying to use environment variables with `ExecTask`, but I can't make it work.

Supposing I have the following `demo.sh` script:

```sh
#!/bin/bash

echo "hello ${MY_VAR}"
``` 

When I execute it with `$ MY_VAR=world ./demo.sh` I have the following output:

```console 
$ MY_VAR=world ./demo.sh 
hello world
```

So I'm trying to do the same with `ExecTask` and `<env>` but it doesn't work.

My buildfile:

```xml
<?xml version="1.0" encoding="UTF-8" ?>

<project name="Testing env" default="say-hello">

    <target name="say-hello">
        <exec executable="./demo.sh" passthru="true">
            <env key="MY_VAR" value="world"/>
        </exec>
    </target>

</project>
```  

This is the output when I execute `phing  -f demo.xml say-hello -debug`:

```console
# (lot of removed stuff here)
Build sequence for target 'say-hello' is: say-hello 
Complete build sequence is: say-hello  

Testing env > say-hello:

BuildEvent[source=say-hello]
BuildEvent[source=UnknownElement]
  +Task: exec
    -calling setter ExecTask::setExecutable()
    -calling setter ExecTask::setPassthru()
    -calling adder ExecTask::addEnv()
    -calling setter EnvVariable::setKey()
    -calling setter EnvVariable::setValue()
     [exec] Current OS is Linux
     [exec] Executing command: ./demo.sh
hello 
BuildEvent[source=UnknownElement]
BuildEvent[source=say-hello]

BUILD FINISHED

Total time: 0.0614 seconds

BuildEvent[source=Project]
```

As you can see the output is simply `hello`, although it should be `hello world`. Please also note the line `[exec] Executing command: ./demo.sh`, shouldn't it have the environment variable (`MY_VAR=world`) prepended?

Am I using `<env>` correctly?

**Comment by siad007** [comment]
@jawira you are totally right. This is a bug and should be fixed asap!
Thanks for reporting.

**Comment by siad007** [comment]
Fixed in #1254
